src/translations/screens/subscreens/TravelToken.ts

```diff
@@ -97,7 +97,7 @@ const SelectTravelTokenTexts = {
     ),
     noMobileToken: _(
       'Vi forsøker å klargjøre din mobil til å kunne bruke billettene på den, men det ser ut som det tar litt tid. Du kan sjekke om du har nettilgang og hvis ikke så vil appen forsøke på nytt neste gang du får tilgang.',
-      'We are trying to make your mobile phone ready to be used for the tickets, but it seems it is taking some time. Check that your device is connected to the Internet and if not the phone will try again when the Internet connection is back.',
+      'We are trying to make your phone ready to be used for the tickets, but it seems it is taking some time. Check that your device is connected to the Internet and if not the phone will try again when the Internet connection is back.',
```

Comment:
@JuliaOppegard Har du et forslag på den setningen her?
Lurer på om jeg heller ville foreslått noe sånt som:

"We're trying to get your phone ready for ticket usage, but it seems like it's taking some time. Check that your device is connected to the internet, if not we'll try again the internet connection is back."

Reply:
Forslag:
"We're trying to get your phone ready for ticket usage, but it seems like it's taking some time. Check that your device is connected to the internet, if not we'll try again once the internet connection is restored.

Reply:
:+1: